addons/l10n_it_edi_sdicoop/views/res_config_settings_views.xml

```diff
@@ -36,6 +36,21 @@
                                 </div>
                             </div>
                         </div>
+                        <div class="col-12 col-lg-6 o_setting_box">
+                            <div class="o_setting_right_pane">
+                                <div class="content-group">
+                                    <span class="o_form_label">
+                                        Fattura Elettronica test mode
+                                    </span>
+                                    <div class="text-muted">
+                                        Odoo will send the invoices to a non-production service for demo purposes.
```

Comment:
This is confusing as above it is says test mode, but below it could say Production.  (and for the moment, we would still be lying)
```suggestion
                                    <span class="o_form_label">
                                        Fattura Elettronica Environment
                                    </span>
                                    <div class="text-muted">
                                        In demo mode, Odoo will send the invoices to a non-production service.  
```

Reply:
I've changed `test mode` to `demo mode` but `Environment` is not really a work that user can understand.

Reply:
@tsb-odoo any comment?

Reply:
At the moment I've put: "In demo mode Odoo will just simulate the sending of invoices to the government."